sdk/storage/storage-queue/swagger/README.md

```diff
@@ -191,3 +195,27 @@ directive:
     transform: >
       $["x-ms-client-name"] = "queueAnalyticsLogging"
 ```
+
+### Update service version from "2018-03-28" to "2019-02-02"
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $.parameters.ApiVersionParameter
+    transform: $.enum = [ "2019-02-02" ];
```

Comment:
Please check if swagger in central repo need be updated to 2019-02-02 for queue service.

Reply:
@tg-msft 
When would we have the swagger for 2019-02-02 version in the Azure/azure-rest-api-specs repo ?